src/ITensorNumericalAnalysis.jl

```diff
@@ -19,7 +19,12 @@ export IndexMap,
   calculate_ind_values,
   dimension,
   dimensions,
-  grid_points
+  grid_points,
+  is_real,
```

Comment:
Base has `isreal`, so we could overload that for our datatypes? There's a bit of question if these should be exported because of the conceptual difference as well (`isreal` applies over elements, do we want it to tell us if there's a tensor with complex entries?)